packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -291,6 +303,8 @@ class WebDevices extends PollingDeviceDiscovery {
       _webServerDevice,
       if (_chromeDevice.isSupported())
         _chromeDevice,
+      if (_edgeDevice.isSupported())
```

Comment:
Does this check if it's a Chromium-based Edge or the old one?

Reply:
I'll add a registry check for the version number

Reply:
Done